go/obscuronode/enclave/enclave.go

```diff
@@ -15,6 +16,13 @@ import (
 
 const ChainID = 777 // The unique ID for the Obscuro chain. Required for Geth signing.
 
+var errRollupNotFound = errors.New("rollup not found")
```

Comment:
This seems to be used only once? Can we use 
`fmt.Errorf("rollup not found: r_%s", parent)` instead ?

Reply:
The linter complains:

`err113: do not define dynamic errors, use wrapped static errors instead: "fmt.Errorf(\"rollup not found: r_%s\", parent)" (goerr113)`

Reply:
Would `errors.New(fmt.Errorf("rollup not found: r_%s", parent))` work ?

Reply:
So if you try that, the compiler complains, because `fmt.Errorf` doesn't return a string.

Reply:
> So if you try that, the compiler complains, because fmt.Errorf doesn't return a string.
Yeah, that's true! 

Since we won't be comparing this error I think it's safe to ignore the linter ?

Reply:
Yeah, go on then.